src/compounds/ActionsGroup/ActionsGroup.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { ListGroup } from 'react-bootstrap'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import SendMessage from './actions/SendMessage'
+import './actions-group.scss'
+
+const ActionsGroup = ({ handleSendingMessages }) => {
+  const [show, setShow] = useState(false)
+  const [action, setAction] = useState(null)
+
+  const handleShow = (componentName) => {
+    setAction(componentName)
+    setShow(true)
+  }
+
+  const handleClose = () => {
+    setAction(null)
+    setShow(false)
+  }
+
+  return (
+    <>
+      <ListGroup className='actions-group'>
+        <ListGroup.Item action variant='primary' onClick={() => handleShow('SendMessage')}>
+          <FontAwesomeIcon icon='fa-envelope' />
+          Send Message
+        </ListGroup.Item>
+      </ListGroup>
+      {(action === 'SendMessage' && show)
+        && (
+          <SendMessage
+            handleClose={handleClose}
+            onSubmit={handleSendingMessages}
+          />
+        )}
```

Comment:
the SendMessage component has to be on the page when ActionsGroup is rendered, or else the onClick in the list group item won't work. 

so it's visibility is always true in the actual component, but we set whether it is showing/hidden from here. 

this allows us to keep SendMessage (and future actions) as their own components without clogging up this file.